src/main/java/com/github/kaklakariada/fritzbox/model/User.java

```diff
@@ -18,13 +18,21 @@
 package com.github.kaklakariada.fritzbox.model;
 
 import org.simpleframework.xml.Attribute;
+import org.simpleframework.xml.Text;
 
 public class User {
 
     @Attribute(name = "last", required = false)
-    public boolean last;
+    private int last;
+
+    @Text
+    private String name;
 
     public boolean isLast() {
```

Comment:
Sorry for nit-picking. But if this returns an int, we should rename the getter.
```suggestion
    public boolean getLast() {
```

Reply:
Yeah, thought about this too. The main thing is that "it looks like it should indicate the last used user". 
Unfortunately the documents on AVM-Site are not updated yet. I will ask AVM-Support for updating the Documents (or if this was a mistake with the latest version). 

I don't see any usecase related to an integer (maybe iam wrong, but "last" and "second last" and so on is a bit weird for me), from my perspective this should be treaten as a boolean.

Reply:
👍